CRM/Contact/Form/Domain.php

```diff
@@ -120,7 +120,14 @@ public function buildQuickForm(): void {
     //build location blocks.
     $this->assign('addressSequence', CRM_Core_BAO_Address::addressSequence());
     CRM_Contact_Form_Edit_Address::buildQuickForm($this, 1);
-    CRM_Contact_Form_Edit_Email::buildQuickForm($this, 1);
+
+    //Email box
+    $this->addField("email[1][email]", [
+      'entity' => 'email',
+      'aria-label' => ts('Email %1', [1 => 1]),
+      'label' => ts('Email %1', [1 => 1]),
```

Comment:
This is an odd `ts` now. I guess it's already translated but there is also already a literal "Email 1" in the translation strings so this could be simplified without adding a new string.

Reply:
https://github.com/civicrm/civicrm-core/pull/29861